pyrit/memory/memory_models.py

```diff
@@ -177,8 +212,8 @@ class EmbeddingDataEntry(Base):  # type: ignore
     __tablename__ = "EmbeddingData"
     # Allows table redefinition if already defined.
     __table_args__ = {"extend_existing": True}
-    id = mapped_column(Uuid(as_uuid=True), ForeignKey(f"{PromptMemoryEntry.__tablename__}.id"), primary_key=True)
-    embedding = mapped_column(ARRAY(Float).with_variant(JSON, "mssql"))  # type: ignore
+    id = mapped_column(SQLiteUUID, ForeignKey(f"{PromptMemoryEntry.__tablename__}.id"), primary_key=True)
+    embedding = mapped_column(ARRAY(Float).with_variant(JSON, "mssql").with_variant(JSON, "sqlite"))  # type: ignore
```

Comment:
is this a mistake? Should it just be:

```suggestion
    embedding = mapped_column(ARRAY(Float).with_variant(JSON, "sqlite"))  # type: ignore
```

Reply:
thanks! fixed